libs/sdk-ui-filters/src/AttributeFilterHandler/types/attributeFilterLoader.ts

```diff
@@ -0,0 +1,134 @@
+// (C) 2022 GoodData Corporation
+import { IAttributeFilter } from "@gooddata/sdk-model";
+import { GoodDataSdkError } from "@gooddata/sdk-ui";
+import {
+    AsyncOperationStatus,
+    CallbackPayloadWithCorrelation,
+    CallbackRegistration,
+    Correlation,
+} from "./common";
+import { IAttributeLoader } from "./attributeLoader";
+import { IAttributeElementLoader } from "./elementsLoader";
+
+/**
+ * Handles the loading of the all required metadata for the attribute filter handler.
+ *
+ * @alpha
+ */
+export interface IAttributeFilterLoader extends IAttributeLoader, IAttributeElementLoader {
+    /**
+     * Get the effective filter.
+     */
+    getFilter(): IAttributeFilter;
+
+    /**
+     * Loads all the required data to initialize the attribute filter handler:
+     * - Attribute
```

Comment:
Should we do the TSDoc workaround for lists? having an empty line between each item so that API reference is rendering that as a list properly?
Same for some other TSDocs in this PR